packages/slider/src/Slider.tsx

```diff
@@ -211,11 +211,8 @@ export class Slider extends React.PureComponent<SliderProps, SliderState> {
       return;
     }
 
-    // Grab the next viable slide index
-    const nextSlideIndex = this.findNext();
```

Comment:
so I'm assuming nextSlideIndex value was getting cached and not being re-evaluated?

Reply:
Yeah